content/docker-hub/download-rate-limit.md

```diff
@@ -1,58 +1,97 @@
+   you can only view your own usage. 
+
+2. In Docker Hub, select **Usage**.
+3. In the drop-down, select whether to download your personal data or
+   data for an organization.
+4. In **From** and **To**, select a date range for the data.
+5. Select **Send report to email** to have Docker email you a link to the data
+   file. Note that email processing time may vary.
 
+The file contains the following comma separated values.
 
-## Definition of limits
+| CSV column         | Definition                                                                                                                                                                                                                 | Usage guidance                                                                                                                                                                |
+|--------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `datehour`           | The date and hour of the pull that resulted in the data transfer.                                                                                                                                                          | This helps in identifying peak usage times and patterns.                                                                                                                      |
+| `user_name`          | The Docker ID of the user that pulled the image                                                                                                                                                                            | This lets organization owners track data consumption per user and manage resources effectively.                                                                               |
+| `repository`         | The name of the repository of the image that was pulled.                                                                                                                                                                   | This lets you identify which repositories are most frequently accessed and consume most of the data transfer.                                                                 |
+| `pat_name`           | Name of the personal access token that was used for authentication with Docker CLI.                                                                                                                                        | Personal access tokens are usually used to authenticate automated tools (Docker Desktop, CI/CD tools, etc.). This is useful for identifying which automated system issued the pull. |
```

Comment:
question: is `generated` == tokens generated by Docker Desktop? (should we call that out here in that case?)